### ISSUE TITLE
[IDEA] add a configuration to define the load paths

### ISSUE THREAD

**Issue by matteosister** [doctrine/DoctrineFixturesBundle#102]
What do you think about adding a configuration to specify the paths from where doctrine should load the fixtures?

By now there is the --fixtures option, but is long to type, and in symfony we can rely on the config component.

This is **just an idea**!...but it's easy to implement....so let me know if I can work on a pr...

**Comment by alcaeus** [comment]
No longer relevant after #209.
